data/remnant/remnant 2 side missions.txt

```diff
@@ -640,3 +640,58 @@ mission "Remnant: Shattered Light 6"
 			`	Taely gestures take a tone of curiosity. "One thing that you could help us with, if you wish, is resources. Right now we do not have any real need to put all this machinery to work, and as such we do not have the resources to spare to get it fully operational. That being said, if ever you would like a replica of the Shattered Light, we could use this equipment to build one."`
 			`	Her gestures take on an apologetic tone. "It will not be a perfect replica as we will have to use our own systems for most of the internals, and the resource allotment cost will be significantly higher than it would have been back at the Tycho Crater shipyard on Luna... but if you have the resources and interest, the opportunity is there. The shipyard will have more information for you in a day or two." Her gestures trail off with a sign that indicates vast possibilities.`
 			`	"Someday, Captain <first>, much may come from this. But for now, it will be months before we have analyzed everything. Thank you for helping recover a piece of our history, Captain."`
+
+
+
+mission "Remnant: Scent of Rain 1"
+	landing
+	name "Search for the unknown Korath ship."
+	description "Chilia has asked you to find the Korath ship that can carry three 'olofez."
```

Comment:
It occurred to me that my previous suggestion of waypointing the three Exile systems would be a bad idea since it'd require visiting all three to complete this without some shenanigans with invisible missions and stuff. This description should give some indication as to where the player should look. It should probably also mention that we need to do an outfit scan. Maybe:
```suggestion
	description "Chilia has asked you to find and conduct an outfit scan of the Korath ship that can carry three 'olofez. The Remnant believe it will be found in one of the southern Korath systems."
```
This format also feels off. I feel like we usually get mission descriptions telling us what to do, possibly with extra information like who asked us to do it. This is telling us that someone has asked us to do something. I don't know how consistent the rest of the game is on that, though.